packages/web5/lib/src/dids/did_web/did_web.dart

```diff
@@ -1,49 +1,117 @@
 import 'dart:convert';
 import 'dart:io';
 
+import 'package:web5/src/crypto.dart';
+import 'package:web5/src/dids/bearer_did.dart';
+import 'package:web5/src/dids/did.dart';
 import 'package:web5/src/dids/did_core.dart';
 import 'package:web5/src/dids/did_method_resolver.dart';
-import 'package:web5/src/dids/did.dart';
 
 class DidWeb {
   static const String methodName = 'web';
-  static final resolver = DidMethodResolver(name: methodName, resolve: resolve);
+
+  static final DidMethodResolver resolver = DidMethodResolver(
+    name: methodName,
+    resolve: resolve,
+  );
+
+  static Future<BearerDid> create({
+    required AlgorithmId algorithm,
+    required KeyManager keyManager,
+    required String url,
+    List<String>? alsoKnownAs,
+    List<String>? controllers,
+    List<DidService>? services,
+    List<DidCreateVerificationMethod>? verificationMethods,
```

Comment:
do you happen to have `DidCreateVerificationMethod` defined somewhere? i can't find it

Reply:
I've added it to this branch so it could be merged before dht-create if wanted